src/libcollections/vec.rs

```diff
@@ -1301,6 +1301,41 @@ impl<T: Clone> Vec<T> {
     }
 }
 
+impl<T: Default> Vec<T> {
```

Comment:
Hmm. I agree, although for now I think it'd be best to continue the pattern and refactor the entire file in a future PR.

Also, this seems like a good candidate for a fmt RFC.